azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServerModule.java

```diff
@@ -28,9 +42,71 @@
  */
 public class AzkabanWebServerModule extends AbstractModule {
 
+  private static final Logger log = Logger.getLogger(AzkabanWebServerModule.class);
+  private static final String USER_MANAGER_CLASS_PARAM = "user.manager.class";
+  private static final String VELOCITY_DEV_MODE_PARAM = "velocity.dev.mode";
+
   @Override
   protected void configure() {
     bind(Server.class).toProvider(WebServerProvider.class);
     bind(AzkabanWebServer.class).in(Scopes.SINGLETON);
+    bind(ScheduleLoader.class).to(TriggerBasedScheduleLoader.class);
+  }
+
+  @Inject
+  @Singleton
+  @Provides
+  public UserManager loadUserManager(final Props props) {
```

Comment:
Some commenting here and for the function below would be useful - especially because most of us aren't guice wizards ⚡️